api4/team_test.go

```diff
@@ -2982,7 +2982,8 @@ func TestInviteUsersToTeam(t *testing.T) {
 	user1 := th.GenerateTestEmail()
 	user2 := th.GenerateTestEmail()
 
-	emailList := []string{user1, user2}
+	memberInvite := &model.MemberInvite{Emails: []string{user1, user2}}
+	emailList := memberInvite.Emails
```

Comment:
Could we add another test to make sure both inputs are still valid?

Reply:
the test is already there https://github.com/mattermost/mattermost-server/pull/19849/files#diff-f157eb201938e9e26eff6958772bfa684b18152b6697e7cc94fb2261ef4f3003R1326

Reply:
In that test you are passing also a MemberInvite model with only emails. We should test that the api endpoint can receive just an array of strings like it did before